src/bootman/bootman.h

```diff
@@ -350,9 +352,35 @@ static inline void kernel_array_free(void *v)
         nc_array_free(&a, (array_free_func)free_kernel);
 }
 
+/**
+ * Get freestanding initrd
```

Comment:
This should include explanation of what "freestanding" means. What it can be used for etc.